api/src/test/java/org/openmrs/api/FormServiceTest.java

```diff
@@ -244,6 +245,7 @@ public void getFormField_shouldNotFailWithNullIgnoreFormFieldsArgument() {
 
 	 */
 	@Test
+	@Disabled("Temporarily until a fix is found DONT MERGE")
```

Comment:
@dkayiwa I wanted to check if we still need this behaviour, for getForms to return duplicates. In master it returns 3 of the same row (same java object). Hibernate changed this behaviour - https://docs.jboss.org/hibernate/orm/6.0/migration-guide/migration-guide.html#query-sqm-distinct

Wanted to check with you before I looked any deeper, as I haven't been able to figure out how to force duplicates yetback again.